docs/docs/explanations/advanced-profiles-settings.md

```diff
@@ -18,6 +18,19 @@ profiles:
       description: Stable environment with 1 cpu / 1 GB ram
       access: all
       default: true
+      profile_options:
+        image:
+          display_name: Image
+          choices:
+            default:
+              display_name: quay.io/nebari/nebari-jupyterlab:latest
+              default: true
+              kubespawner_override:
+                image: quay.io/nebari/nebari-jupyterlab:latest
+            custom:
+              display_name: <my-container-registry>/myOrg/my-custom-image:mytag
+              kubespawner_override:
+                image: <my-container-registry>/myOrg/my-custom-image:mytag
```

Comment:
It will be nice to add a screenshot for this too, below the default screenshot as that would make it much clear on what to expect on the UI as a user.

Reply:
The screenshot is updated in [f3b71ad](https://github.com/nebari-dev/nebari-docs/pull/559/commits/f3b71ad69a08b1e45fb53a24862c5bde0823ae73)